sass/sass.bzl

```diff
@@ -218,3 +218,79 @@ sass_binary = rule(
     attrs = _sass_binary_attrs,
     outputs = _sass_binary_outputs,
 )
+
+def _multi_sass_binary_impl(ctx):
+  """multi_sass_binary accepts a list of sources and compile all in one pass.
+
+  Args:
+    ctx: The Bazel build context
+
+  Returns:
+    The multi_sass_binary rule.
+  """
+
+  inputs = depset([f for t in ctx.attr.srcs for f in t.files])
+  outputs = []
+  for f in inputs:
+    if f.basename.startswith("_"):
+      continue
+    name = _strip_extension(f.basename)
+    outputs.append(ctx.actions.declare_file(
+      name + ".css",
+      sibling = f,
+    ))
+    if ctx.attr.sourcemap:
+      outputs.append(ctx.actions.declare_file(
+        name + ".css.map",
+        sibling = f,
+      ))
+
+  args = ctx.actions.args()
+  args.add("--style", ctx.attr.output_style)
+  if not ctx.attr.sourcemap:
+    args.add("--no-source-map")
+  root = ctx.label.package
+  args.add("--load-path", root)
+  args.add(root + ":" + ctx.bin_dir.path + '/' + root)
+
+  ctx.actions.run(
+    inputs = inputs,
+    outputs = outputs,
+    executable = ctx.executable.compiler,
+    arguments = [args],
+    mnemonic = "SassCompiler",
+    progress_message = "Compiling Sass",
+  )
+
+  return [DefaultInfo(files = depset(outputs))]
+
+multi_sass_binary = rule(
+  implementation = _multi_sass_binary_impl,
+  attrs = {
+    "srcs": attr.label_list(
+      doc = "A list of stylesheets to compile",
```

Comment:
"stylesheets" -> "Sass files and associated assets"

Reply:
done